scurl/cgurl.pyx

```diff
@@ -62,6 +62,10 @@ cdef bytes build_netloc(char * url, Parsed parsed):
     else:
         raise ValueError
 
+cdef urljoin_fallback(char * base, char * url, bool allow_fragments, bool decode):
+    if decode:
+        return stdlib_urljoin(base, url, allow_fragments=allow_fragments).decode('utf-8')
+    return stdlib_urljoin(base, url, allow_fragments=allow_fragments)
```

Comment:
Would be nice to remove ``stdlib_urljoin(base, url, allow_fragments=allow_fragments)`` repetition, in case some changes are needed here.

Reply:
hmm, so what do you think I should put here as a fallback method instead?

Reply:
@nctl144 I meant something like this, to avoid spelling the long call two times, sorry for such a minor note:
```
    joined = stdlib_urljoin(base, url, allow_fragments=allow_fragments)
    return joined.decode('utf-8') if decode else joined
```